### PR TITLE
DOCSP-9605: issues and help page

### PR DESCRIPTION
**Pull request by ccho-mongodb**
### Issue JIRA link:
https://jira.mongodb.org/browse/DOCSP-9605

### Docs staging link (requires sign-in on MongoDB Corp SSO):
https://docs-mongodbcom-staging.corp.mongodb.com/ffeb6f2/java/docsworker-xlarge/DOCSP-9605-issues-help/issues-and-help

### CODE REVIEW

File: source/issues-and-help.txt [hunk 1]
```diff
@@ -4,4 +4,51 @@ Issues & Help
...
+documentation for more information.
+
+Bugs / Feature Requests
+-----------------------
+
+If you think you've found a bug or want to see a new feature in the Java
+driver, please open a case in our issue management tool, JIRA:
+
+
+
```
Comment (nathan-contino-mongo) [inline source/issues-and-help.txt:21]: Could trim these extra newlines.

File: source/issues-and-help.txt [hunk 2]
```diff
@@ -4,4 +4,51 @@ Issues & Help
 
 .. default-domain:: mongodb
 
-Lorem ipsum
+We are lucky to have a vibrant MongoDB Java community with lots of varying
+experience of using the Java driver. We often find the quickest way to get
```
Comment (nathan-contino-mongo) [inline source/issues-and-help.txt:8]: Considering that we actively avoid using "we" in most of our documentation, this seems slightly out of place. That being said I don't actually mind it myself, and I do think it mostly works here (especially if you haven't noticed its absence everywhere else in our docs)

Comment (nathan-contino-mongo) [inline source/issues-and-help.txt:8]:
```suggestion
experiences of using the Java driver. We often find the quickest way to get
```

Something about this first sentence just doesn't seem right to me. Open to this change or a reword.

Comment (ccho-mongodb) [reply source/issues-and-help.txt:8]: Actively avoiding "we" is news to me. Is there a specific reason why?

Comment (nathan-contino-mongo) [reply source/issues-and-help.txt]: I think it's more that we avoid first-person subjective. Probably worth bringing up in the team chat if it doesn't line up with your own understanding of our style -- I could be wrong!

File: source/issues-and-help.txt [hunk 3]
```diff
@@ -4,4 +4,49 @@ Issues & Help
 
 .. default-domain:: mongodb
 
-Lorem ipsum
+We are lucky to have a vibrant MongoDB Java community that includes users
+with varying levels of experience using the Java driver. We find the quickest
```
Comment (nathan-contino-mongo) [inline source/issues-and-help.txt:8]: nice change, this reads much more naturally.

File: source/issues-and-help.txt (hunk 2 above)
Comment (ccho-mongodb) [reply source/issues-and-help.txt]: I don't mind being fluid on writing style, but there has to be a good reason. I know that Nathan and Nick specifically endorsed the usage of "we", especially when providing a reader directions, when I asked about preferred pronouns last year. I'm also wondering if this is a recent decision that I wasn't part of as I'm fairly certain our authoring team for Node Driver Guidance Docs has all used "we" in the documentation.

Comment (nathan-contino-mongo) [reply source/issues-and-help.txt]: I don't think anything has changed on this recently. Probably just a pre-coffee moment for me.

**Pull request merged**